### PR TITLE
feat: add minio adapter for logs

### PR DESCRIPTION
**Pull request by nicufk**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: cmd/logs/main.go [hunk 1]
```diff
@@ -55,7 +55,14 @@ func main() {
 		WithGrpcAddress(cfg.GrpcAddress)
 
 	// TODO - add adapters here
-	svc.AddAdapter(adapter.NewDummyAdapter())
+	minioAdapter, err := adapter.NewMinioAdapter(cfg.MinioEndpoint, cfg.MinioAccessKey, cfg.MinioSecretKey, cfg.MinioRegion, cfg.MinioToken, cfg.MinioBucket)
+	if err != nil {
+		log.Errorw("error creating minio adapter", "error", err)
+		svc.AddAdapter(adapter.NewDummyAdapter())
```
Comment (vsukhin) [inline cmd/logs/main.go:61]: do we need DummyAdapter in prod?

Comment (vsukhin) [reply cmd/logs/main.go:61]: sounds like, log service will not store logs without minio?

**Review by exu — APPROVED** [review]

File: cmd/logs/main.go (hunk 1 above)
Comment (exu) [reply cmd/logs/main.go:61]: Yeah I would rename it to LogAdapter or sth and attach it onlly when debug mode is set

**Review by vsukhin — APPROVED** [review]
a couple of questions

File: cmd/logs/main.go (hunk 1 above)
Comment (exu) [reply cmd/logs/main.go:61]: Good point - it should only try to set minio when it's not in agent mode

Comment (exu) [reply cmd/logs/main.go:61]: I wold put whole logic into if when in OSS mode
additionally add additional dummyadapter when DEBUG is set 
and add cloud adapter when Agent key is set

Comment (nicufk) [reply cmd/logs/main.go:61]: got it, I will make the changes then

**Pull request merged**